main.py

```diff
@@ -2,6 +2,29 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    (Actor.objects.filter(first_name="George").filter(last_name="Klooney")
+     .update(last_name="Clooney"))
```

Comment:
brackets are redundant. Also no need to use two filters. Fix in all places
```suggestion
    Actor.objects.filter(
        first_name="George", last_name="Klooney"
    ).update(last_name="Clooney"))
```